docs-chef-io/content/workstation/config_rb.md

```diff
@@ -16,11 +16,7 @@ aliases = ["/config_rb.html", "/config_rb_knife.html", "/config_rb/"]
 
 {{< warning >}}
 
-The `config.rb` file is a replacement for the knife.rb file, starting
-with the Chef Client 12.0 release. The `config.rb` file has identical
-settings and behavior to the knife.rb file. Chef Infra Client looks
-first for the presence of the `config.rb` file and if it is not found,
-then looks for the `knife.rb` file.
+The `config.rb` file is a replacement for the knife.rb file. The `config.rb` file has identical settings and behavior to the knife.rb file. Chef Infra Client looks first for the presence of the `config.rb` file and if it is not found, then looks for the `knife.rb` file.
```

Comment:
knife.rb in the first sentence needs backticks

Reply:
Fixed